### PR TITLE
Upgrade Isolates for GraalVM Native Images Demo.

### PR DESCRIPTION
**Pull request by LesiaChaban**

### CODE REVIEW

File: native-netty-plot/pom.xml [hunk 1]
```diff
@@ -8,20 +8,19 @@
...
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <graalvm.version>21.0.0.2</graalvm.version>
   </properties>
 
   <dependencies>
     <dependency>
-      <groupId>org.graalvm.nativeimage</groupId>
-      <artifactId>native-image-maven-plugin</artifactId>
-      <version>${graalvm.version}</version>
+     <groupId>org.graalvm.buildtools</groupId>
```
Comment (olyagpl) [inline native-netty-plot/pom.xml:15]: @LesiaChaban, this is not the way to specify the Maven plugin for Native Image building. See how to https://graalvm.github.io/native-build-tools/latest/maven-plugin-quickstart.html > step 6.

File: native-netty-plot/README.md [hunk 2]
```diff
@@ -35,7 +36,7 @@ The example is built with Maven:
 This creates a JAR file with all dependencies embedded: `target/netty-plot-0.1-jar-with-dependencies.jar`.
 
 The [pom.xml](pom.xml) specifies the expected dependencies to Netty, emp4j, and JFreeSVG.
-It also specifies a dependency on Native Image, `org.graalvm.nativeimage`, - a technology to ahead-of-time compile Java code into a standalone executable.
+It also specifies a dependency on Native Image, `native-image-maven-plugin` for building and testing GraalVM Native Images, a technology to ahead-of-time compile Java code into a standalone executable.
```
Comment (olyagpl) [inline native-netty-plot/README.md:39]: This sentence doesn't make much sense.
Replace with:
```
There is also the [Maven plugin for GraalVM Native Image building](https://graalvm.github.io/native-build-tools/latest/index.html) to compile this Java application into a native executable.
```

**Review by olyagpl — CHANGES_REQUESTED** [review]

File: native-netty-plot/pom.xml [hunk 3]
```diff
@@ -67,10 +130,10 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.11.0</version>
           <configuration>
-              <source>21</source>
-              <target>21</target>
+              <source>1.8</source>
```
Comment (olyagpl) [inline native-netty-plot/pom.xml:133]: why did you revert to Java 8 version?

File: native-netty-plot/pom.xml [hunk 4]
```diff
@@ -33,6 +35,67 @@
...
+                                    <goal>compile-no-fork</goal>
+                                </goals>
+                                <phase>package</phase>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <imageName>${imageName}</imageName>
+                            <fallback>false</fallback>
+                            <buildArgs>
+                                <buildArg>-H:DashboardDump=fortune -H:+DashboardAll</buildArg>
```
Comment (olyagpl) [inline native-netty-plot/pom.xml:62]: Lesia, this is the demo for Native Image isolates, why the name is "fortune" and why do we generate the data for GraalVM dashboard?

**Review by olyagpl — CHANGES_REQUESTED** [review]

File: native-netty-plot/pom.xml (hunk 3 above)
Comment (LesiaChaban) [reply native-netty-plot/pom.xml]: I have already changed it to:
```
<source>${java.version}</source>
<target>21</target>
```

File: native-netty-plot/pom.xml (hunk 4 above)
Comment (LesiaChaban) [reply native-netty-plot/pom.xml]: It was my copy paste mistake. I fixed it!

**Pull request merged**